### ISSUE TITLE
An error will occur when use function `accuracy`

### ISSUE THREAD

**Issue by TaiseiYamana** [thuml/Transfer-Learning-Library#20]
I clone it with google colab and run it.
You will always get an error like this.

Do you get a similar error if you do the same?

<img width="1581" alt="スクリーンショット 2021-01-02 13 09 34" src="https://user-images.githubusercontent.com/54575368/103450565-c45f1500-4cfb-11eb-9559-583f82c386f0.png">

**Comment by JunguangJiang** [comment]
Sorry, the link to this image is lost and I cannot see your errors.

**Comment by JunguangJiang** [comment]
Sorry, it seems that the erros is caused by the version of pytorch. I used pytorch1.4.0 one year ago. In the lastest pytorch version, I met the same mistake.  My solution is to modify accuracy function to

```
def accuracy(output, target, topk=(1,)):
    with torch.no_grad():
        maxk = max(topk)
        batch_size = target.size(0)

        _, pred = output.topk(maxk, 1, True, True)
        pred = pred.t()
        correct = pred.eq(target[None])

        res = []
        for k in topk:
            correct_k = correct[:k].flatten().sum(dtype=torch.float32)
            res.append(correct_k * (100.0 / batch_size))
        return res
```

Hope it works for you!

**Comment by JunguangJiang** [comment]
Also, you might need to change 
```
top1.update(acc1[0], images.size(0))
top5.update(acc5[0], images.size(0))
```
to
```
top1.update(acc1.item(), images.size(0))
top5.update(acc5.item(), images.size(0))
```
in the validate function.
